### PR TITLE
Do not display inlay hints as bold

### PR DESCRIPTION
**Pull request by mrnugget**
I think bold is the least fitting font weight for inlay hints, which should be subtle hints and not, well, bold.

If someone feels strongly about this, I can revert, but only if we add the ability to change this per theme.

Until then: beautiful, thin, subtle inlay hints!

Release Notes:

- Improved styling of inlay hints by not making them bold in the editor.

screenshot-2024-02-23-17 30 29@2x

### CODE REVIEW

File: assets/themes/gruvbox/gruvbox.json [hunk 1]
```diff
@@ -482,7 +482,7 @@
         "hidden": "#998b78ff",
         "hidden.background": "#4c4642ff",
         "hidden.border": "#544c48ff",
-        "hint": "#8c957dff",
+        "hint": "#6a695bff",
```
Comment (maxdeviant) [inline assets/themes/gruvbox/gruvbox.json:485]: Just making sure: is this change intentional?

Comment (mrnugget) [reply assets/themes/gruvbox/gruvbox.json:485]: Yes, 100% :P

**Comment by mikayla-maki** [comment]
Excited for this :D
